apps/masterbots.ai/components/routes/chat/chat-panel-header.tsx

```
@@ -32,26 +31,25 @@ export function ChatPanelHeader({
-                  <div className="flex items-center space-x-1 drop-shadow-lg">
-                    <div className="rounded-full size-2 bg-primary animate-pulse" />
-                    <div className="rounded-full size-2 bg-primary animate-pulse" />
-                    <div className="rounded-full size-2 bg-primary animate-pulse" />
-                  </div>
-                  <p className="text-sm font-bold text-primary drop-shadow-lg">
-                    {loadingState}
-                  </p>
+              <div className="flex items-center justify-between gap-4">
+                <div className="flex items-center space-x-1 drop-shadow-lg">
+                  {Array.from({ length: 3 }).map((_, index) => (
+                    <span key={index} className="animate-pulse rounded-full bg-primary size-1" style={{ animationDelay: `${index * 100}ms` }} />
```

Biome

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)